augraphy/augmentations/lowinkline.py

```diff
@@ -105,23 +93,56 @@ def add_transparency_line(self, mask, y, alpha=None):
                     )[0]
 
         else:
-            low_ink_line = self.inconsistent_transparency_line(mask[y, :])
+            low_ink_line = (np.random.random((xsize)) * 255).astype("uint8")
+            if len(mask.shape) > 2:
+                new_low_ink_line = np.zeros((1, xsize, mask.shape[2]), dtype="uint8")
```

Comment:
In the previous code, noises are added randomly in 3 channels of mask and this created random colour in the created lines:

![image](https://user-images.githubusercontent.com/52521979/202894578-584f39d5-db01-414a-825f-ce207718c23b.png)

This is fixed in the current commit by assigning all channels with a same noise:
![image](https://user-images.githubusercontent.com/52521979/202894628-9a392aef-8fe5-4ec9-8bd7-f2175ca8b2dc.png)